Sample/Generated/Lucid/Support/CoreManagerContainer.swift

```diff
@@ -128,12 +136,12 @@ extension CoreManagerContainer: RelationshipCoreManaging {
         switch entityType {
         case GenreIdentifier.entityTypeUID:
             return genreManager.get(
-                byIDs: DualHashSet(identifiers.lazy.compactMap { $0.toRelationshipID() }),
+                byIDs: identifiers.lazy.compactMap { $0.toRelationshipID() }.uniquified(),
```

Comment:
Oh wow, we *really* neglected this project, haha. These changes are from a while back.